builtin/mainmenu/tab_online.lua

```diff
@@ -17,14 +17,31 @@
 
 local function get_sorted_servers()
 	local servers = {
+		lan = {},
 		fav = {},
 		public = {},
 		incompatible = {}
 	}
 
+	local merged_serverlist = table.copy(serverlistmgr.servers)
+
+	--Special thanks to proller <proller@gmail.com> for letting use use the get_lan_servers function.
+	if minetest.settings:get_bool("serverlist_lan") then
```

Comment:
Are citations like this needed? They feel out of place.

Reply:
it's your pull request no?